doc/ansible/user-guide.md

```diff
@@ -264,15 +264,20 @@ $ docker push quay.io/example/memcached-operator:v0.0.1
 
 Kubernetes deployment manifests are generated in `deploy/operator.yaml`. The
 deployment image in this file needs to be modified from the placeholder
-`REPLACE_IMAGE` to the previous built image. To do this run:
+`REPLACE_IMAGE` to the previous built image.
+Also, we need to update the `imagePullPolicy` to `Always` (makes it easier for testing).
+
+To do the above run:
 ```
-$ sed -i 's|REPLACE_IMAGE|quay.io/example/memcached-operator:v0.0.1|g' deploy/operator.yaml
+$ sed -i 's|image:.*|image: quay.io/example/memcached-operator:v0.0.1|g' deploy/operator.yaml
```

Comment:
Why you change this one?

Reply:
Thee current sed command works perfect.

Reply:
So, could you please revert these ones?

Reply:
Hey @camilamacedo86 ,

Thanks for looking into it. Please see the below explanation and let me know if I have missed anything. I had to make these changes to make things work in `v0.15.2`

The `REPLACE_IMAGE` is an Ansible variable. If we just replace it with a value, Ansible will throw an error as `quay.io/example/memcached-operator:v0.0.1` is not a variable.
```
~/.../memcached-operator (ansible) $ grep REPLACE_IMAGE deploy/operator.yaml 
          image: "{{ REPLACE_IMAGE }}"
          image: "{{ REPLACE_IMAGE }}"
~/.../memcached-operator (ansible) $ sed -i 's|REPLACE_IMAGE|quay.io/example/memcached-operator:v0.0.1|g' deploy/operator.yaml
~/.../memcached-operator (ansible) $ grep "image:" deploy/operator.yaml 
          image: "{{ quay.io/example/memcached-operator:v0.0.1 }}"
          image: "{{ quay.io/example/memcached-operator:v0.0.1 }}"
~/.../memcached-operator (ansible) $
```

And we need to have a value against `imagePullPolicy`, otherwise Kubernetes will throw this error.
```bash
~/.../memcached-operator (ansible) $ kubectl apply -f deploy/operator.yaml
The Deployment "memcached-operator" is invalid: 
* spec.template.spec.containers[0].imagePullPolicy: Unsupported value: "{{ pull_policy|default('Always') }}": supported values: "Always", "IfNotPresent", "Never"
* spec.template.spec.containers[1].imagePullPolicy: Unsupported value: "{{ pull_policy|default('Always') }}": supported values: "Always", "IfNotPresent", "Never"
~/.../memcached-operator (ansible) $
```